test/checker.test.ts

```diff
@@ -23,52 +23,56 @@ describe('useChecker', () => {
     vi.clearAllMocks()
   })
 
-  it('logs an error', async () => {
-    const mockValidator = vi.fn().mockImplementation(() => ({ valid: false, results: [] }))
-    const { checkHTML: checker } = useChecker({ validateString: mockValidator } as any)
-
-    await checker('https://test.com/', '<a><a>Link</a></a>')
-    expect(mockValidator).toHaveBeenCalled()
-    expect(console.error).toHaveBeenCalled()
-  })
-
   it('prints an error when invalid html is provided', async () => {
     const mockValidator = vi.fn().mockImplementation(() => ({ valid: false, results: [] }))
     const { checkHTML: checker } = useChecker({ validateString: mockValidator } as any, false)
 
     await checker('https://test.com/', '<a>Link</a>')
     expect(mockValidator).toHaveBeenCalled()
-    expect(console.error).toHaveBeenCalled()
+    expect(console.log).not.toHaveBeenCalled()
+    expect(console.warn).not.toHaveBeenCalled()
+    expect(console.error).toHaveBeenCalledWith(
+      `HTML validation errors found for ${chalk.bold('https://test.com/')}`
+    )
   })
 
-  it('prints a warning when invalid html is provided and log level is set to verbose', async () => {
-    const mockValidator = vi.fn().mockImplementation(() => ({ valid: true, results: [{ messages: [{ message: '' }] }] }))
+  it('prints an error when invalid html is provided and log level is set to verbose', async () => {
+    const mockValidator = vi.fn().mockImplementation(() => ({ valid: false, results: [] }))
```

Comment:
ie. isn't this what we are wanting to test - that the overall result is valid but that there are still warnings to show?